Bottom: 812ea89cc72f642645cd9220838ffe4638c50c30
Top:    665fbcb4d0ead2916b172eb293099fcdcc3c5cb1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:55:52 +0000

draw.h: Remove unused DrawHoles() prototype


---

diff --git a/src/draw.h b/src/draw.h
index e0688ee..bae0cd8 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -70,7 +70,6 @@ void DrawLayerGroup (int side, const BoxType *drawn_area);
 void DrawPaste (int side, const BoxType *drawn_area);
 void DrawSilk (HID_DRAW *hid_draw, int side, const BoxType *drawn_area);
 void DrawMask (int side, const BoxType *drawn_area);
-void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
 void PrintAssembly (int side, const BoxType *drawn_area);
 
 /* TEMPORARY */
